App/Redux/RootReducer.ts

```diff
@@ -17,6 +17,7 @@ export default combineReducers({
   contacts: contactsReducer,
   devices: devicesReducer,
   preferences: prefrencesReducer,
+  notifications: notificationsReducer,
```

Comment:
Cool, think you need to add the 'notifications' key to the persistence whitelist as well.

Reply:
done